config.toml

```diff
@@ -110,7 +110,7 @@ algolia_docsearch = true
 # current release branch - could be rc
 release_branch = "main"
 # the main version. Never is rc.
-release_version = "v1.18.0"
```

Comment:
keep this 1.18 only We wll change this we release 1.19.

Reply:
done